cmd/cephcsi.go

```diff
@@ -78,7 +78,7 @@ func getType() string {
 
 func getDriverName() string {
 	// was explicitly passed a driver name
-	if driverName != nil && len(*driverName) != 0 {
+	if driverName != nil && *driverName != "" {
```

Comment:
@Madhu-1 length check (len(..) )looks to be the better method

Reply:
issue found by golangci-lint